src/components/Animal/Animal.spec.js

```diff
@@ -0,0 +1,116 @@
+import Animal from './Animal';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+describe('Single Animal-Display', () => {
+	it("renders every important detail of animal & the names of spots where animal lives & NOT the names of spots where animal is not living  & the names of organizers who arrange diving with animal & NOT the names of organiZers who don't arrange diving with animal ", () => {
```

Comment:
I think it is really hard to use this test in real life, i.e. if it fails to figure out what's wrong.
I would separate it in different `it` statements (what is rendered and what is not); you can create variables above with your fake data which you use in line 9 for the `spots` prop (and the other props as well) and thus hand over this variable every time your Animal component is rendered in the different `it` statements.
By doing so, you don't have to write the fake data for every single `it`.